spec/services/event_spec.rb

```diff
@@ -19,7 +18,6 @@
         subject.trigger(:reticulated_splines,
                         foo: "Bar",
                         pi: 3.14,
-                        baz: [1, "string", 0.5],
```

Comment:
And replace this with `baz: [1, "string", 0.5].to_s` (whatever that is)?

Reply:
yep